src/test/java/pl/byczazagroda/trackexpensesappbackend/controller/WalletRestControllerTest.java

```diff
@@ -0,0 +1,37 @@
+package pl.byczazagroda.trackexpensesappbackend.controller;
+
+import lombok.SneakyThrows;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.jupiter.api.Test;
+import org.junit.runner.RunWith;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.http.MediaType;
+import org.springframework.test.context.TestPropertySource;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+import pl.byczazagroda.trackexpensesappbackend.BaseControllerTest;
+import pl.byczazagroda.trackexpensesappbackend.model.dto.CreateWalletDto;
+import pl.byczazagroda.trackexpensesappbackend.service.WalletServiceImpl;
+
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@RunWith(SpringJUnit4ClassRunner.class)
+@TestPropertySource("classpath:data-test.properties")
+class WalletRestControllerTest extends BaseControllerTest {
+
+  @MockBean
+  WalletServiceImpl walletService;
+
+  @SneakyThrows
+  @Test
+  void shouldReturnCreatedCodeCodeWhenWalletCreated() {
+
+    CreateWalletDto createWalletDto = new CreateWalletDto("Test wallet name");
+
+    mockMvc.perform(post("/api/v1/wallets")
```

Comment:
I think, that you should create empty tests (same as service, implementation and controller) which's always successful